src/pycram/orm/task.py

```diff
@@ -15,14 +15,14 @@ class TaskTreeNode(Base):
     id: Mapped[int] = mapped_column(autoincrement=True, primary_key=True, init=False)
     """id overriden in order to be able to set the remote_side of the parent attribute"""
 
-    code_id: Mapped[int] = mapped_column(ForeignKey("Code.id"), default=None)
+    code_id: Mapped[int] = mapped_column(ForeignKey("Code.id"), init=False)
     code: Mapped["Code"] = relationship(init=False)
-    start_time: Mapped[datetime.datetime] = mapped_column(default=None)
-    end_time: Mapped[Optional[datetime.datetime]] = mapped_column(default=None)
```

Comment:
Keep an eye here that this has changed in the PR from me. So redo this after it has been merged.